app/Models/User.php

```
@@ -310,32 +311,43 @@ public function isInstanceAdmin()
-    }
+        if (is_null($sessionTeamId)) {
+            return null;
+        }
 
-    public function otherTeams()
-    {
-        return Auth::user()->teams->filter(function ($team) {
-            return $team->id != currentTeam()->id;
+        return Cache::remember('team:'.$this->id, 3600, function () use ($sessionTeamId) {
+            return Team::find($sessionTeamId);
         });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'currentTeam'. (undefined)

(StaticAccess)